Resolve syncing conflicts from repo_sync_working_branch to public

openpublishbuild opened the pull request:
Pull request opened by Docs to resolve syncing conflicts from repo_sync_working_branch to public.

nithinnara committed:
Update advanced-spam-filtering-asf-options.md

Remove the deprecation warning since we have no plans for it. We just want to discourage its use as it can be a very aggressive approach that can cause FPs. But we already have a warning for this.

markjjo committed:
new epic connector topic

markjjo committed:
Merge branch 'master' into markjjo-EpicConnector

markjjo committed:
formatting, acronlinx fixes

nithinnara committed:
Update advanced-spam-filtering-asf-options.md

Fixing misleading text

markjjo committed:
Merge branch 'master' into markjjo-EpicConnector

markjjo committed:
fixed field table

markjjo committed:
Merge branch 'master' into markjjo-EpicConnector

chrisda committed:
Update preset-security-policies.md

Updates for BIP

markjjo committed:
Merge branch 'master' into markjjo-EpicConnector

markjjo committed:
New topic for healthcare connector

jmgriffee committed:
Create compliance-easy-trials-compliance-manager-assessments.md

jmgriffee committed:
Update compliance-manager-templates.md

jmgriffee committed:
Update TOC.yml

jmgriffee committed:
Merge branch 'master' into jgriffee-cm-working

jmgriffee committed:
Update compliance-easy-trials-compliance-manager-assessments.md

markjjo committed:
Merge branch 'master' into markjjo-EpicConnector

markjjo committed:
completed draft healthcare connector article

jmgriffee committed:
cm updates

jmgriffee committed:
Update compliance-manager-templates.md

jmgriffee committed:
Update compliance-easy-trials-compliance-manager-assessments.md

jmgriffee committed:
Update compliance-easy-trials-compliance-manager-assessments.md

jmgriffee committed:
Update compliance-easy-trials-compliance-manager-assessments.md

jmgriffee committed:
Update compliance-easy-trials-compliance-manager-assessments.md

jmgriffee committed:
Update compliance-manager-assessments.md

jmgriffee committed:
m365 tweaks

jmgriffee committed:
Update compliance-manager-templates-list.md

jmgriffee committed:
Merge branch 'master' into jgriffee-cm-working

jmgriffee committed:
Merge branch 'master' into jgriffee-cm-working

chrisda committed:
Merge branch 'master' into BIP-chrisda

chrisda committed:
Merge branch 'master' into BIP-chrisda

chrisda committed:
Updates for BIP

jmgriffee committed:
Update compliance-manager-whats-new.md

jmgriffee committed:
Merge branch 'master' into jgriffee-cm-working

jmgriffee committed:
Update compliance-manager-whats-new.md

chrisda committed:
Quarantine notification updates

jmgriffee committed:
link name changes

Priyankag16 committed:
update system requirement for mde android

Priyankag16 committed:
add what's new page for android mde

jmgriffee committed:
Merge branch 'master' into jgriffee-cm-working

Priyankag16 committed:
update section to configure mde

daisyfell committed:
new steps and images

daisyfell committed:
Merge branch 'master' into vl-sharepoint-df

daisyfell committed:
metadata add

faoquong committed:
Update quarantine-policies.md

@chrisda

Priyankag16 committed:
Merge branch 'master' into priyanka-mdeandroidupdates

Priyankag16 committed:
Merge branch 'master' into priyanka-mdeandroidupdates

mjcaparas committed:
Update microsoft-defender-endpoint-android.md

mjcaparas committed:
Update android-whatsnew.md

mjcaparas committed:
Update TOC.md

mjcaparas committed:
Update android-configure.md

mjcaparas committed:
Update android-whatsnew.md

Commit:
minor wording update

mjcaparas committed:
Update android-whatsnew.md

Commit:
Merge branch 'master' of https://github.com/MicrosoftDocs/microsoft-365-docs-pr into cabailey-m365docs-cloudattachments

mjcaparas committed:
Update android-configure.md

mjcaparas committed:
Merge branch 'master' into priyanka-mdeandroidupdates

garycentric committed:
Merge branch 'master' into vl-sharepoint-df

Commit:
remove erroneous text

mjcaparas committed:
Update android-whatsnew.md

Commit:
add links to checklists

chrisda committed:
Merge pull request #11293 from faoquong/patch-8

Update quarantine-policies.md

mjcaparas committed:
Merge pull request #11295 from Priyankag16/priyanka-mdeandroidupdates

Priyanka mdeandroidupdates

chrisda committed:
Merge branch 'master' into Quarantine-chrisda

chrisda committed:
Merge branch 'Quarantine-chrisda' of https://github.com/MicrosoftDocs/microsoft-365-docs-pr into Quarantine-chrisda

chrisda committed:
Merge branch 'master' into Quarantine-chrisda

garycentric committed:
Added image border by applying the newer style of image ref

When "type" is "content", an image border is applied

garycentric committed:
Merge branch 'master' into vl-sharepoint-df

garycentric committed:
Merge pull request #11291 from MicrosoftDocs/vl-sharepoint-df

Vl sharepoint df

chrisda committed:
Merge branch 'master' into patch-2

garycentric committed:
Merge branch 'master' into cabailey-m365docs-cloudattachments

chrisda committed:
Merge pull request #11106 from nithinnara/patch-2

Remove ASF deprecation warning

garycentric committed:
Merge branch 'master' into cabailey-m365docs-cloudattachments

garycentric committed:
Merge pull request #11298 from cabailey/cabailey-m365docs-cloudattachments

minor wording update

garycentric committed:
Merge branch 'master' into jgriffee-cm-working

chrisda committed:
Update advanced-spam-filtering-asf-options.md

chrisda committed:
Merge branch 'master' into patch-3

chrisda committed:
Merge pull request #11145 from nithinnara/patch-3

Update advanced-spam-filtering-asf-options.md

chrisda committed:
ASF is not being deprecated

Related changes from #11106

chrisda committed:
Merge branch 'master' into Quarantine-chrisda

chrisda committed:
Quarantine notification updates

In ancillary topics

garycentric committed:
Merge branch 'master' into jgriffee-cm-working

Dansimp committed:
updating metadata

chrisda committed:
Update quarantine-policies.md

Clarified default quarantine policies used (DefaultFullAccessPolicy vs. NotificationEnabledPolicy)

chrisda committed:
Merge pull request #11300 from MicrosoftDocs/chrisda

ASF is not being deprecated

chrisda committed:
Merge branch 'master' into Quarantine-chrisda

Dansimp committed:
fixing borkn metadata

chrisda committed:
Merge pull request #11301 from MicrosoftDocs/Quarantine-chrisda

Quarantine-chrisda to Master

Dansimp committed:
fixing broken metadata 2

garycentric committed:
Merge pull request #11302 from MicrosoftDocs/master

Publish 11/01/2021, 3:30 PM

Dansimp committed:
fix

Commit:
Merge branch 'master' into dansimp-metadata4

Commit:
Merge pull request #11303 from MicrosoftDocs/dansimp-metadata4

Dansimp metadata4

Commit:
Update manage-updates-baselines-microsoft-defender-antivirus.md

Commit:
Update manage-updates-baselines-microsoft-defender-antivirus.md

Commit:
Update manage-updates-baselines-microsoft-defender-antivirus.md

Commit:
Update manage-updates-baselines-microsoft-defender-antivirus.md

Commit:
Update manage-updates-baselines-microsoft-defender-antivirus.md

Commit:
Update manage-updates-baselines-microsoft-defender-antivirus.md

denisebmsft committed:
Merge pull request #11305 from MicrosoftDocs/mdav-relnotes

Mdav relnotes

denisebmsft committed:
Merge pull request #11306 from MicrosoftDocs/master

pushing updates live

chrisda committed:
Clarified that quarantine policies do not control the retention period

Per email request

Commit:
Update manage-updates-baselines-microsoft-defender-antivirus.md

officedocspr commented:
## ![STOP](https://docs.microsoft.com/en-us/office/media/internal/prfilecountcheck-stop-sign.png)  Caution: Large pull request

This **public** repo applies limits on the number of files that can be merged in a single pull request (PR):

- **Warning** Warning message and ability to unblock PR using the *Merge large pull request*<sup>\*</sup> label - 30
- **Block** Block message and PR is blocked - 100

This pull request will change **69** files, which exceeds the **warning** limit of **30**.

Before this pull request can be merged, you must review the list of files it contains and their changes to confirm they are correct and intentional. **This includes files you did not change and/or do not own**. If there are changes you aren't familiar with, contact the individuals who submitted the commits included in this pull request and review the changes with them.

**The individual who merges this pull request is responsible for ensuring the changes included in it are correct, and for resolving any issues that might result in merging it.**

### To merge this pull request

If you have determined that the changes in this pull request are correct, add the **Merge large pull request**<sup>\*</sup> label to it, wait for the **max/pr-file-count** check to pass, and then click **Merge**. To add a label, click the gear icon next to **Labels** and then select **Merge large pull request**<sup>\*</sup>.

![Label UI screenshot](https://docs.microsoft.com/en-us/office/media/internal/prfilecountcheck-label-screenshot.gif)

### To reject this pull request

If you can't confirm the changes in this pull request are correct, click **Close pull request** at the bottom of this page to close it without merging. Work with your team to revert any changes that aren't correct.

<sup>\*</sup> You must have write or triage access to this repo to add labels.

denisebmsft committed:
Merge pull request #11308 from MicrosoftDocs/mdav-relnotes

Update manage-updates-baselines-microsoft-defender-antivirus.md

chrisda committed:
Merge branch 'master' into Quarantine-chrisda

denisebmsft committed:
Merge pull request #11309 from MicrosoftDocs/master

pushing updates live

chrisda committed:
Merge pull request #11307 from MicrosoftDocs/Quarantine-chrisda

Clarified that quarantine policies do not control the retention period

chrisda committed:
Update anti-spam-protection.md

Missing quarantine policy update

chrisda committed:
Merge branch 'master' into Quarantine-chrisda

officedocspr commented:
## ![STOP](https://docs.microsoft.com/en-us/office/media/internal/prfilecountcheck-stop-sign.png)  Caution: Large pull request

This **public** repo applies limits on the number of files that can be merged in a single pull request (PR):

- **Warning** Warning message and ability to unblock PR using the *Merge large pull request*<sup>\*</sup> label - 30
- **Block** Block message and PR is blocked - 100

This pull request will change **69** files, which exceeds the **warning** limit of **30**.

Before this pull request can be merged, you must review the list of files it contains and their changes to confirm they are correct and intentional. **This includes files you did not change and/or do not own**. If there are changes you aren't familiar with, contact the individuals who submitted the commits included in this pull request and review the changes with them.

**The individual who merges this pull request is responsible for ensuring the changes included in it are correct, and for resolving any issues that might result in merging it.**

### To merge this pull request

If you have determined that the changes in this pull request are correct, add the **Merge large pull request**<sup>\*</sup> label to it, wait for the **max/pr-file-count** check to pass, and then click **Merge**. To add a label, click the gear icon next to **Labels** and then select **Merge large pull request**<sup>\*</sup>.

![Label UI screenshot](https://docs.microsoft.com/en-us/office/media/internal/prfilecountcheck-label-screenshot.gif)

### To reject this pull request

If you can't confirm the changes in this pull request are correct, click **Close pull request** at the bottom of this page to close it without merging. Work with your team to revert any changes that aren't correct.

<sup>\*</sup> You must have write or triage access to this repo to add labels.

chrisda committed:
Merge branch 'Quarantine-chrisda' of https://github.com/MicrosoftDocs/microsoft-365-docs-pr into Quarantine-chrisda

chrisda committed:
Merge branch 'master' into Quarantine-chrisda

chrisda committed:
Merge branch 'master' into BIP-chrisda

markjjo committed:
Merge branch 'master' into markjjo-EpicConnector

garycentric committed:
Merge branch 'master' into repo_sync_working_branch

markjjo committed:
Merge pull request #11311 from MicrosoftDocs/markjjo-EpicConnector

New Epic and Healthcare connectors for Ignite

garycentric committed:
Correct capitalization in 34 file names

garycentric committed:
Correct spelling error in image reference

garycentric committed:
Correct format of "> [!NOTE]"

garycentric committed:
Correct format of note

garycentric committed:
Correct formatting of 3 notes

garycentric committed:
Correct font weight of table headings

Table headings are bold by default. Adding formatting for bold results in a lighter weight font than is standard on the platform.

garycentric committed:
Move "Steps" into nearest column heading

"Steps" was marked up so that the Markdown processor would consider a fifth heading for this four-column table. I moved "Steps" to be part of the fourth column, "Configuration", and deleted the spaces at the end of the line.

garycentric committed:
Replace non-functional bookmarks with file names

garycentric committed:
Merge branch 'master' into repo_sync_working_branch

garycentric committed:
Correct formatting of 2 notes

chrisda committed:
Merge branch 'master' into Quarantine-chrisda

chrisda committed:
Merge pull request #11310 from MicrosoftDocs/Quarantine-chrisda

Quarantine-chrisda to Master

chrisda committed:
Merge branch 'master' into BIP-chrisda

chrisda committed:
BIP updates

chrisda committed:
Merge branch 'master' into BIP-chrisda

chrisda committed:
Merge pull request #11312 from MicrosoftDocs/BIP-chrisda

BIP-chrisda to Master

garycentric committed:
Add lightbox to illegible images

garycentric committed:
Merge branch 'master' into repo_sync_working_branch

garycentric committed:
Merge pull request #11314 from MicrosoftDocs/v-gmoor-fix-pr-11290

Correct links, file names, markup, and presentation in PR 11290

garycentric committed:
Minimum changes to reach Acrolinx score of 80

garycentric committed:
Merge pull request #11315 from MicrosoftDocs/v-gmoor-fix-pr-11290

Minimum change to reach Acrolinx score of 80 in diagnosing-performance-issues-with-sharepoint-online.md

garycentric committed:
Merge pull request #11290 from MicrosoftDocs/repo_sync_working_branch

Confirm merge from repo_sync_working_branch to master to sync with https://github.com/MicrosoftDocs/microsoft-365-docs (branch public)

yogkumgit commented:
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

Commit:
update links

Commit:
update links

cabailey committed:
Merge pull request #11320 from cabailey/cabailey-m365docs-updates

update links

v-dihans committed:
Merge branch 'master' into jgriffee-cm-working

v-dihans committed:
Merge pull request #11287 from MicrosoftDocs/jgriffee-cm-working

Compliance Manager: new trial page + wording edits (for morning of 11/2)

v-dihans committed:
Merge pull request #11324 from MicrosoftDocs/master

OOB Publish for https://github.com/MicrosoftDocs/microsoft-365-docs-pr/pull/11287

officedocspr commented:
## ![STOP](https://docs.microsoft.com/en-us/office/media/internal/prfilecountcheck-stop-sign.png)  Too many files - PR blocked

This **public** repo applies limits on the number of files that can be merged in a single pull request (PR):

- **Warning** Warning message and ability to unblock PR using the *Merge large pull request*<sup>\*</sup> label - 30
- **Block** Block message and PR is blocked - 100

This pull request will change **130** files, which exceeds the **block** limit of **100**. **This PR will not be merged**.

### To merge this pull request

You must either reduce the number of files being changed to below the 100 file limit or split the changes into multiple PRs. PRs must not have more than 100 files and fewer than 30 files if possible.

### To reject this pull request

If you want to abandon these changes and not merge this PR, click **Close pull request** at the bottom of this page to close it without merging. Work with your team to revert any changes that aren't correct.

<sup>\*</sup> You must have write or triage access to this repo to add labels.
